CONTRIBUTING.md

```diff
@@ -1,263 +1,63 @@
-- Education: Is there an alternative solution? You can name it even if you don't reject the review because of it.
-
-It is the responsibility of the reviewee to make sure that the code is easy to review. Some more background: 
-- [Gitlab's code review guidelines](https://docs.gitlab.com/ee/development/code_review.html) has some good bullets on how to do code reviews.
-- [thoughtbot code review guide](https://github.com/thoughtbot/guides/tree/master/code-review)
-
-## Using GIT
-
-### Development workflow with GIT: Branching strategy
-
-**Development on feature branches, merge to master**:
-- Develop the code in the branch created for the issue
-- Once the code is ready to be merged with master, resolve WIP status and create a new pull request
-- Ask one of your peers to review the code and merge the pull request
-- Any feature added to the project should include unit tests. If anyone modifies the features that are already there, tests should be refined accordingly. Finally if a bug is found, a test should be written that makes sure this bug would be detected in the future
-
-###  Commit Messages
+Documentation is a very crucial part of the project, because it ensures usability of the package. We develop the docs in the following way:
 
-Before making commits, make sure that you have [https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup](configured your name and email)
-
-When creating commit messages, keep these guidelines in mind. They are meant to keep the messages useful and relevant.
-
-- Include a short description of *what* and *why* was done, *how* can be seen in the code.
-- Use present tense, imperative mood
-- Limit the length of the first line to 72 chars. You can use multiple messages to specify a second (longer) line: `git commit -m "Patch load function" -m "This is a much longer explanation of what was done"`
-
-To help you with this message format, you can have git generate new commit messages with template by running:
-
-`git config commit.template .gitmessage`
-
-Example message:
-```
-Create feature X
-This is a longer description of what the commit contains. See #123
-```
-
-## Issue tracking
-
-Issue tracking is recommended even for solo projects. In this project we use Gitlab issue board for issue tracking.
-
-
-## Versioning and Deployment
-
-In the project we use versioning based on Semantic Versioning for Python. After a number of changes made to the repository, we make a deployment.
-
-Deployment is made in the following steps:
-- Merge a MR with the following changes
-    - Bump the version in setup.py
-    - Bump release variable in docs/source/conf.py
-    - Update Changelog with changes made from previous version
-- Create a git tag for the version. This will trigger a CI pipeline which deploys 
-- Deployment of new version is performed by Maintainers. 
+* `docs/` folder contains all the relevant documentation
+* We use `mkdocs serve` to view the documentation locally.
```

Comment:
You can add:
`Use it to test the documentation, every time you make any changes to it.`